api/v1beta1/types.go

```diff
@@ -147,6 +147,16 @@ type NetworkSpec struct {
 	// +kubebuilder:default:=1460
 	// +optional
 	Mtu int64 `json:"mtu,omitempty"`
+
+	// MinPortsPerVM: Minimum number of ports allocated to a VM from this NAT
+	// config. If not set, a default number of ports is allocated to a VM. This is
```

Comment:
Nit: we might want to specify that this is configured on the Nat Router, I wasn't entirely sure what this doing otherwise.  But it's only a comment, so not a blocker IMO.